.eleventy.js

```diff
@@ -7,6 +7,8 @@ const markdownItAnchor = require('markdown-it-anchor');
 const { headingLinks } = require("./config/headingLinks");
 const yaml = require("js-yaml");
 const svgSprite = require("eleventy-plugin-svg-sprite");
+const syntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight");
```

Comment:
Adding this package so we can have code syntax highlighting! Seems to work pretty well but we can always mess with the template if we don't like it.